packages/math-input/src/components/input/math-input.tsx

```diff
@@ -328,11 +336,21 @@ class MathInput extends React.Component<Props, State> {
 
     blur: () => void = () => {
         this.mathField.blur();
-        this.props.onBlur && this.props.onBlur();
-        this.setState({focused: false, handle: {visible: false}});
+
+        this.setState((prevState) => {
+            return {
+                showInputFocusStyle: false,
+                handle: {
+                    visible: false,
+                },
+                focused: prevState.focused,
+            };
+        });
```

Comment:
I don't think the callback is needed. If you don't pass `focused` React should just let it be.

```suggestion
        this.setState({
                showInputFocusStyle: false,
                handle: {
                    visible: false,
                },
        });
```